OutSystems/outsystems-datagrid-reactive
# ROU-3513: Adding ToggleRowDragging client action

This PR adds ToggleRowDragging client action.

### What was done
* Renamed ColumnReorder feature to GridReorder.
* Created an auxiliary function to help build our API responses.
* Created two new helper functions to help parse/stringify JSON maps.

### Test Steps
1. Enable Reorder row
2. Reorder rows 1 and 2

Expected: Rows should be reordered.

#

1. Enable Reorder row
2. Reorder rows 1 and 2
3. Press Save layout
4. Reload page
5. Press Apply layout

Expected: rows should be reordered.

#

1. Enable Reorder row
2. Sort any column
3. Reorder rows 1 and 2

Expected: An error message should show on the console and rows should not be reordered

#

1. Enable Reorder row

Expected: An error message should show on the console and rows should not be reordered.

#

1. Enable Reorder row and column reordering
2. Reorder rows 1 and 2
3. Reorder columns 1 and 2

Expected: Rows and columns should be reordered.

#

1. Enable Reorder row
2. Disable column reorder
3. Reorder rows 1 and 2
4. Reorder columns 1 and 2

Expected: Rows should be reordered and columns should not be reordered.
